addons/website_event_sale/models/event_event.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from odoo import _, api, models
+
+
+class Event(models.Model):
+    _name = 'event.event'
+    _inherit = 'event.event'
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        if self.env.context.get('from_website_content_form'):
+            for vals in vals_list:
```

Comment:
Wait, what ? No ?

Reply:
Hello @tde-banana-odoo 

I am no python or even Odoo-orm expert but this solution does not seem horrible enough to me for this kind of comment. @xO-Tx actually talked to me about it and I did not think of another alternative (the context stuff) (well maybe there is better to do at the model level or whatever but this definitely does not seem worse than what was there before?)

If you are talking about the loop through `vals_list`, it does not seems necessary indeed (@xO-Tx aren't you building the same object to add in all `vals`?).

Reply:
Yep, I wanted to explain more but we also have our PRs to r+, sorry forgot to post remaining content :D . Too much things to do these days ...

Actually, do we need that code ? Either events are created from templates, either from default values. But it seems strange to me to create dummy tickets like that (especially with 1000 seats, why 1000) ?

I know this code existed before your changer, but it actually feels meh to me. Especially doing it at create level, even with a context key is strange to me.

Final question: why doing it in stable ? Can't we cleanup in master and see if we can get rid of event code instead ?

Reply:
> Actually, do we need that code ?

Same, we have our own PRs to r+, I did not spent time questioning the features of other apps TBH. With such big refactoring, the priority is to keep to behaviors we can keep as a first step :grimacing: (in this case, restore it as it was lost in the initial merge). If you say you prefer to drop it, we can drop it, master/16.0 lives without it since a few months now.

> Final question: why doing it in stable ?

This commit just removes **unused controllers**, seems reasonable to still do it in 16.0 to me. Indeed it was restoring that event ticket stuff that was lost when the controller stopped being used too, but we can simply drop it (as unused) if you prefer.

Reply:
@tde-banana-odoo Any news? Is Luc ok to remove the code? :+1:

Reply:
Hello,

Yes, let us go for removing it. Rationale (to be included in commit somehow probably): events default values are managed by templates. Moreover we have links in frontend to edit the even when being allowed to edit it (event groups), notably when there is no ticket in registration page. So it makes no sense to create dummy tickets, without price, for 1000 seats. Those values are random so they will have to be edited. Better let users create real tickets if necessary, than editing / removing dummy tickets.

Cheers !